### PR TITLE
Feature/shubham/sentry

### PR DESCRIPTION
**Pull request by panic-coder**

### CODE REVIEW

File: notification-service/index.js [hunk 1]
```diff
@@ -10,6 +10,15 @@ let initRedis = require('./utils/initRedis');
 const logger = require('sp-json-logger')({fileName: + 'notification-service' + fileName});
 let SSE = require('express-sse');
 let authUsers = require('./middleware/authUsers');
+const Sentry = require('@sentry/node');
+var sentryData = process.env.STOCKUP_WEB
```
Comment (kamal0808) [inline notification-service/index.js]: choose better variable name

File: notification-service/package.json [hunk 2]
```diff
@@ -14,6 +14,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@sentry/node": "^5.10.2",
```
Comment (kamal0808) [inline notification-service/package.json]: lock this version

File: package.json [hunk 3]
```diff
@@ -4,6 +4,7 @@
   "dependencies": {
     "@mean-expert/loopback-component-realtime": "1.0.0-rc.9.6",
     "@mean-expert/loopback-sdk-builder": "2.3.1",
+    "@sentry/node": "^5.10.2",
```
Comment (kamal0808) [inline package.json]: lock this version

File: server/server.js [hunk 4]
```diff
@@ -9,7 +9,17 @@ var boot = require('loopback-boot');
 //     http://docs.strongloop.com/display/public/LB/Working+with+LoopBack+objects
 var app = module.exports = loopback();
 var path = require('path');
+const Sentry = require('@sentry/node');
+var sentryData = process.env.STOCKUP_WEB;
+console.log("sentryData : ",sentryData);
```
Comment (kamal0808) [inline server/server.js]: use logger

File: workers/package.json [hunk 5]
```diff
@@ -4,6 +4,7 @@
   "description": "Background workers for the Stockup webapp",
   "main": "server.js",
   "dependencies": {
+    "@sentry/node": "^5.10.2",
```
Comment (kamal0808) [inline workers/package.json]: lock version

File: workers/sqsWorker.js [hunk 6]
```diff
@@ -10,6 +10,17 @@ var clearRequire = require('clear-require');
 var SUCCESS = 0;
 var FAILURE = 1;
 
+const Sentry = require('@sentry/node');
+var sentryData = process.env.STOCKUP_WORKER_V2;
```
Comment (kamal0808) [inline workers/sqsWorker.js]: choose better variable name

File: workers/syncWorker.js [hunk 7]
```diff
@@ -15,6 +15,18 @@ const SUCCESS = 0;
 const FAILURE = 1;
 const syncInterval = 300;
 
+const Sentry = require('@sentry/node');
+var sentryData = process.env.STOCKUP_WORKER_V3;
```
Comment (kamal0808) [inline workers/syncWorker.js]: choose better variable

File: notification-service/index.js (hunk 1 above)
Comment (panic-coder) [reply notification-service/index.js]: done

File: notification-service/package.json (hunk 2 above)
Comment (panic-coder) [reply notification-service/package.json]: done

File: package.json (hunk 3 above)
Comment (panic-coder) [reply package.json]: done

File: server/server.js (hunk 4 above)
Comment (panic-coder) [reply server/server.js]: done

File: workers/package.json (hunk 5 above)
Comment (panic-coder) [reply workers/package.json]: done

File: workers/sqsWorker.js (hunk 6 above)
Comment (panic-coder) [reply workers/sqsWorker.js]: done

File: workers/syncWorker.js (hunk 7 above)
Comment (panic-coder) [reply workers/syncWorker.js]: done

**Pull request merged**